src/models/likeDao.js

```diff
@@ -0,0 +1,39 @@
+const database = require("./dataSource");
+
+const likeEx = async (id, store_id) => {
```

Comment:
naming을 정학히 할 필요가 있어보입니다. CRUD와 관련하여 동사 (get, create, delete, update)를 활용하여 지어주시면 좋을 것 같습니다!
```suggestion
const getLikesByUserIdAndStoreId = async (id, store_id) => {
```